gitpod-io/website
# [Self-Hosted] Highlight that the external database name needs to be `gitpod`

## Description
As of right now, the name of a database needs to be `gitpod`. We do not make this clear in our documentation and this has impacted customers. 

## How to test
- sense check changes in files


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2678"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2678"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
